turbo/app/make_app_zkevm.go

```diff
@@ -33,15 +32,6 @@ func MakeApp_zkEvm(name string, action cli.ActionFunc, cliFlags []cli.Flag) *cli
 			cli.ShowAppHelpAndExit(context, 1)
 		}
 
```

Comment:
This handling caused always exit for unsupported flags, hence the code for error handling in cdk-erigon/main.go was never called.